unit-tests/live/image-quality/test-image-quality.py

```diff
@@ -172,10 +172,20 @@ def is_depth_meaningful(config, laser_enabled=True, save_image=False, show_image
 ################################################################################################
 
 test.start("Testing depth frame - laser ON -", dev.get_info(rs.camera_info.name))
-res, laser_black_pixels = is_depth_meaningful(cfg, laser_enabled=True, save_image=DEBUG_MODE, show_image=DEBUG_MODE)
+res = False
+max_black_pixels = float('inf')
+for i in range(5):
```

Comment:
please add a cooment why we do this loop and maybe use a const global variable for `FRAMES_TO_CHECK = 5`?

Reply:
Did it really help? The 5 loop?
Do you think it is stable?
We can increase to 30
But we need it 100% stable

Reply:
It can help, mostly it helps if there's no depth found, but it's best to find the frame with the least black pixels anyway.
From what I've seen it is stable - if there is no depth the test will fail, if there is, they both should pass